tests/Aspire.Hosting.Azure.Tests/AzureBicepProvisionerTests.cs

```diff
@@ -250,14 +250,14 @@ public async Task ResourceWithSameScopeHaveSameChecksums()
 
         var parameters0 = new JsonObject();
         var scope0 = new JsonObject();
-        await BicepProvisioner.SetParametersAsync(parameters0, bicep0.Resource);
-        await BicepProvisioner.SetScopeAsync(scope0, bicep0.Resource);
+        await BicepProvisioner.SetParametersAsync(parameters0, bicep0.Resource, cancellationToken: TestContext.Current.CancellationToken);
+        await BicepProvisioner.SetScopeAsync(scope0, bicep0.Resource, TestContext.Current.CancellationToken);
         var checkSum0 = BicepProvisioner.GetChecksum(bicep0.Resource, parameters0, scope0);
 
         var parameters1 = new JsonObject();
         var scope1 = new JsonObject();
-        await BicepProvisioner.SetParametersAsync(parameters1, bicep1.Resource);
-        await BicepProvisioner.SetScopeAsync(scope1, bicep1.Resource);
+        await BicepProvisioner.SetParametersAsync(parameters1, bicep1.Resource, cancellationToken: TestContext.Current.CancellationToken);
+        await BicepProvisioner.SetScopeAsync(scope1, bicep1.Resource, TestContext.Current.CancellationToken);
```

Comment:
with respect to my q on the dashboard PR, line 269 here is an example of a test specific ct. 

What is the correct approach in that case, assuming the goal is to have a shorter timeout than the test default. Should it be combined `CancellationTokenSource.CreateLinkedTokenSource(TestContext.Current.CancellationToken, new CancellationTokenSource(TimeSpan.FromMinutes(5)).Token).Token` (surely there's a better way) ... thoughts? It seems odd for tests like `NestedChildResourcesShouldGetUpdated()` to not get `TestContext.Current.CancellationToken)`